include/nuttx/clock.h

```diff
@@ -394,6 +394,49 @@ void clock_resynchronize(FAR struct timespec *rtc_diff);
 clock_t clock_systime_ticks(void);
 #endif
 
+/****************************************************************************
+ * Name: clock_time2ticks
+ *
+ * Description:
+ *   Return the given struct timespec as systime ticks
```

Comment:
```suggestion
 *   Return the given struct timespec as systime ticks.
```
nit: Missing period.